content/administration/upgrade/odoo_sh.rst

```diff
@@ -1,46 +1,72 @@
 
-=======================
-Odoo.sh version upgrade
-=======================
+=======
+Odoo.sh
+=======
 
-Download and Upload your database
-=================================
+Test upgrade request
+====================
 
-Download a dump of your database (from the :ref:`Builds view <odoosh-gettingstarted-builds-download-dump>`), choose the
-exact copy and without filestore options. Upload the .sql.gz dump on https://upgrade.odoo.com/upload and
-select the Testing Purpose. If you have custom code, you can choose to have it upgraded by us, or do it yourself. Once
-it's processed, you'll get a dump of the database in return.
+Download a dump of your database (from the :ref:`Builds view <odoosh-gettingstarted-builds-download-dump>`),
+choose the **exact copy and without filestore** options. Upload the .sql.gz dump on https://upgrade.odoo.com/upload and
+select the testing purpose. Once it's processed, you'll get a dump of the database in return.
```

Comment:
- 100th character line break
- separate the options in bold
- markup for the .sql.gz file extension

```suggestion
Download a dump of your database (from the :ref:`Builds view
<odoosh-gettingstarted-builds-download-dump>`), select the **exact copy** and **without filestore**
options. Upload the ``.sql.gz`` dump on https://upgrade.odoo.com/upload and select the testing
purpose. Once it's processed, you'll get a dump of the database in return.
```